jobs/sync-integration-tests/templates/pre-start.erb

```diff
@@ -1,18 +1,24 @@
-#!/usr/bin/env bash
+#!/bin/sh
+<% require 'shellwords' %>
 
 set -o errexit
 set -o nounset
 
-PATH="/var/vcap/packages/cli/bin:${PATH}"
+PATH=$PATH:/var/vcap/packages/cli/bin
```

Comment:
Nit. Should keep the `"`-quotes.

Reply:
Also, not sure if the changed order makes a true difference. I believe it should not.